lib/ansible/modules/system/rc_service.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+
+# (c) Mattias Lindvall
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community',
+}
+
+DOCUMENTATION = '''
+module: rc_service
+author:
+    - "Mattias Lindvall"
+version_added: "2.4"
+short_description: Manages services.
+description:
+    - Manages services with rc style init scripts, and sysrc based configuration.
+options:
+    name:
+        required: true
+        description:
+            - Name of the service.
+        aliases: ['unit', 'service']
+    state:
+        required: false
+        default: null
+        choices: ['started', 'stopped', 'restarted']
+        description:
+            - C(started)/C(stopped) are idempotent actions that will not run commands unless necessary.
+              C(restarted) will always restart the service.
+    enabled:
+        required: false
+        default: null
+        choices: ['yes', 'no']
+        description:
+            - Whether the service should be enabled using sysrc.
+requirements:
+    - A system with rc style init scripts, and sysrc based configuration.
+'''
+
+EXAMPLES = '''
+- name: Enable nginx
+  rc_service: name=nginx enabled=yes
+
+- name: Start nginx
+  rc_service: name=nginx state=started
+
+- name: Start and enable nginx
+  rc_service: name=nginx enabled=yes state=stopped
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.sysrc import get_sysrc_value_bool, sysrc_update
+
+
+def run_service(module, args):
+    bin_path = module.get_bin_path('service', required=True)
```

Comment:
I've had 8 and 9 installs w/o it ... but that was mostly cause previous admin avoided it as a 'linuxism', we do have the code for rc, but we can always add later of someone protests. 

so ignore this point for now.